src/comments.js

```diff
@@ -381,6 +383,20 @@ function handleConditionalExpressionComments(enclosingNode, followingNode, comme
   return false;
 }
 
+function handleAssignmentPatternComments(enclosingNode, comment) {
```

Comment:
Good to know, thanks! But this definitely has an impact... Your thoughts on the handleAssignmentPatternComments function, is that the wrong approach?